common/src/main/java/com/wynntils/mc/mixin/LocalPlayerMixin.java

```diff
@@ -49,4 +50,23 @@ private void playSoundPre(SoundEvent sound, float volume, float pitch, CallbackI
             ci.cancel();
         }
     }
+
+    @Inject(
+            method = "Lnet/minecraft/client/player/LocalPlayer;sendPosition()V",
+            at = {
+                @At(
+                        value = "NEW",
+                        target = "(DDDFFZ)Lnet/minecraft/network/protocol/game/ServerboundMovePlayerPacket$PosRot;"),
+                @At(
+                        value = "NEW",
+                        target = "(DDDZ)Lnet/minecraft/network/protocol/game/ServerboundMovePlayerPacket$Pos;"),
+            },
+            cancellable = true)
+    private void sendPositionPre(CallbackInfo ci) {
+        PlayerMoveEvent event = new PlayerMoveEvent();
```

Comment:
This strikes me a bit as odd, that we have a event for moving that does not even include the new location we're moving to.

I wonder if we really need to do this. Afaict, this is used to update the Territory info in Discord. I think we might just as well check the current territory every second or so, and update Discord if it has changed. This looks like it add unnecessary overhead in a lag-sensitive part of the game.